### PR TITLE
Add bash scripts for Schrodinger queue

### PR DESCRIPTION
**Pull request by jugoetz**

### CODE REVIEW

**Comment by jugoetz** [comment]
@Ohyeon5 :P

File: bash-utils/queueligprep [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# script to submit Schrödinger commands using filename from "queue.txt"
+# only submits new jobs when current joblist is shorter than a threshold
+# execute this script frequently through cronjob to use it as queueing system
+# example crontab
+# * * * * * cd /path/to/log_out_dir && /path/to/queueligprep
+
+SCHRODINGER=/opt/schrodinger/suites2023-2
+WORKDIR=/Users/julian/SciAdv\ revisions
```
Comment (Ohyeon5) [inline bash-utils/queueligprep:10]: nit: what about `WORKDIR=PATH_TO_YOUR_DIR` and add a comment saying `# remember to change WORKDIR to where your 'queue.txt' is in`

File: bash-utils/queueqikprop [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# script to submit Schrödinger commands using filename from "queue.txt"
+# only submits new jobs when current joblist is shorter than a threshold
+# execute this script frequently through cronjob to use it as queueing system
+# example crontab:
+# * * * * * cd /path/to/output/dir && /path/to/queueqikprop
+
+SCHRODINGER=/opt/schrodinger/suites2023-2
+WORKDIR=/Users/julian/SciAdv\ revisions/qikprop_out
```
Comment (Ohyeon5) [inline bash-utils/queueqikprop:10]: nit: same as above

**Review by Ohyeon5 — COMMENTED** [review]
@jugoetz Feel free to adapt the suggestions!

File: bash-utils/queueligprep (hunk 1 above)
Comment (jugoetz) [reply bash-utils/queueligprep]: changed defaults + added comments in new commit. The command line argument idea is nice, but I think overkill for this simple script.

File: bash-utils/queueqikprop (hunk 2 above)
Comment (jugoetz) [reply bash-utils/queueqikprop]: as above

**Review by Ohyeon5 — APPROVED** [review]
looks good, thanks!

**Pull request merged**